app/assets/stylesheets/era_custom.css.scss

```diff
@@ -488,6 +488,19 @@ body.home h4{
 	color: $link1Hover;
 }
 
+.avalon>a{
```

Comment:
Just for the footer. I should qualify that in the stylesheet. The styles immediately preceding this one are for a copyright class that is styled the same. Since that class isn't appropriate for the avalon link, I created a new style.

I have to confess that my SASS skills are minimal, but what you posted above looks a lot cleaner/DRYer. Will update.